tests/test_aside.py

```diff
@@ -166,17 +190,28 @@ def test_responses(self):
             course_key=course_id,
             problem_usage_key=UsageKey.from_string(unicode(problem_id) + "extra"),
             user_id=user.id,
-            answer_id='answer_0',
-            answer_text='Answer #0',
+            answer_id=answer_data[0]['answer_id'],
+            answer_text=answer_data[0]['answer_text'],
             event={}
         )
 
-        resp = self.aside_instance.responses()
+        request = RequestFactory().request()
+        request.user = self.instructor
+        request.session = request.environ
+        store = modulestore()
+        problem = store.get_item(problem_id)
+        problem.runtime = self.runtime
+        problem.xmodule_runtime = self.runtime
```

Comment:
1. Seems like it would make sense for the 4 lines above should be a method in `RuntimeEnabledTestCase` (`get_problem_by_id` or something)
1. I'm not clear on why this problem exists in the modulestore in the context of this test case. Whatever this is depending on, it would probably be helpful to make that dependency more explicit